schema/dataKeys.json

```diff
@@ -252,6 +252,11 @@
         "pipelineImage": {
           "type": "string",
           "description": "An image with CLI tools needed for the signing by the internal signing pipelines"
+        },
+        "requestType": {
+          "type": "string",
+          "description": "The request type to be used to request signing",
```

Comment:
Do we really want users to specify this? We already have issues where they use the wrong pyxis certs based on what cluster they are on (-internal not on the internal cluster, no -internal when they are). Isn't this just a determination of if you are on an internal cluster with access to the internal systems? Can't we look at another key (like the pyxis secret) to determine this?

Reply:
in time, we can use the same type of merge script that is used for prod sec properties. They are injected into the RPA before the apply. This script would look at the cluster the RPA is for (already available as an env var) and determine if the cluster is private and inject/overwrite the field.

Right now, I want to be able to select RPAs to enable this feature until battle-tested.